template.yaml

```diff
@@ -232,6 +232,22 @@ Resources:
               - method.request.querystring.page
               - method.request.querystring.results
 
+  NvaCristinListOrganizationProjectsFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: cristin-project
+      Handler: no.unit.nva.cristin.projects.ListCristinOrganizationProjectsHandler::handleRequest
+      Events:
+        NvaListOrganizationProjectsEvent:
+          Type: Api
+          Properties:
+            RestApiId: !Ref NvaCristinProxyApi
+            Path: /organization/{id}/projects
```

Comment:
in openapi you have called it `identifier` . Here it is `id` . Not sure if it matters

Reply:
It should be identifier.. Id is the URI..